package collision_map_creator_msgs.msgs;
import "vector2d.proto";

message CollisionMapRequest
{
  required gazebo.msgs.Vector2d   lowerleft  = 1;
  required gazebo.msgs.Vector2d   upperright = 2;
  required double                 height     = 3;
  required double                 resolution = 4;
  optional string                 filename   = 5 [default = ""];
  optional int32                  threshold  = 6 [default = 255];
  optional string                 groundEntityName = 7 [default = "ground_plane::link::collision"];
  optional double                 minHeight = 8 [default = 0];
}
